src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -240,38 +222,23 @@ public async Task<(IStorageHistory, IStorageItem)> CreateAsync(IStorageItemWithP
                                         string.Join(" ", args.Skip(1)).Replace("%1", source.Path),
                                         PathNormalization.GetParentDir(source.Path)))
                                 {
-                                    status = AppServiceResponseStatus.Success;
+                                    success = true;
                                 }
                             }
                         }
                         else
-                        {
-                            (status, response) = await connection.SendMessageForResponseAsync(new ValueSet()
-                            {
-                                { "Arguments", "FileOperation" },
-                                { "fileop", "CreateFile" },
-                                { "filepath", source.Path },
-                                { "template", newEntryInfo?.Template },
-                                { "data", newEntryInfo?.Data }
-                            });
-                        }
+                            (success, response) = await FileOperationsHelpers.CreateItemAsync(source.Path, "CreateFile", newEntryInfo?.Template, Convert.ToBase64String(newEntryInfo?.Data));
```

Comment:
I don't think the `Convert.ToBase64String` is necessary. We can pass the `byte[]` directly without serialization.